### PR TITLE
Version 3.37.1 part 1

### PR DESCRIPTION
**Pull request by OverOrion**
This is the first batch, another batch is coming soon.

### CODE REVIEW

File: Content/Guides/syslog-ng-guide-admin/tlsoptions.htm [hunk 1]
```diff
@@ -97,6 +97,54 @@ <h6 name="tls-options-ecdh-curve-list"><a name="tls-options-ecdh-curve-list"></a
...
+                        <td class="TableStyle-RuledTableWithHeading_DoNotEdit-BodyA-Column1-Body1">none </td>
+                    </tr>
+                </tbody>
+                <col class="TableStyle-RuledTableWithHeading_DoNotEdit-Column-Column1" style="width: 0.3in;">
+                </col>
+            </table>
+            <p><i style="font-style: normal;">Description:</i> A colon-separated list that specifies the supported signature algorithms associated with client authentication for TLSv1.2 and TLSv1.3, for example, <span class="Code" oldrole="userinput">RSA-PSS+SHA256:ed25519</span>.</p>
+            <p>For servers the value is used in the <pre>signature_algorithms</pre> field of a <pre>CertificateRequest</pre> message.</p>
+            <p>For clients it is used to determine which signature algorithm to use with the client certificate.</p>
+        </div>
```
Comment (MrAnno) [inline Content/Guides/syslog-ng-guide-admin/tlsoptions.htm:147]: This seems to be duplicated.

File: Content/Guides/syslog-ng-guide-admin/parser-mariadb-audit.htm [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <h1 name="parser-mariadb-audit">MariaDB parser</h1>
+        <MadCap:keyword term="segmenting messages">
+        </MadCap:keyword>
+        <MadCap:keyword term="mariadb mariadb-audit parser">
+        </MadCap:keyword>
+        <MadCap:keyword term="mariadb mariadb-audit">
+        </MadCap:keyword>
+        <MadCap:keyword term="parsers:['mariadb-audit']">
+        </MadCap:keyword>
+        <p>The MariaDB parser can parse the log messages of the MariaDB Audit Plugin. The parser supports the <span class="Code" oldrole="parameter">syslog</span> output types's format. Available in version <MadCap:conditionaltext MadCap:conditions="General.OSE">3.37</MadCap:conditionaltext> and later.</p>
```
Comment (tamaskiss00) [inline Content/Guides/syslog-ng-guide-admin/parser-mariadb-audit.htm:16]: oldrole="parameter" not needed

File: Content/Guides/syslog-ng-guide-admin/parser-mariadb-audit.htm [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        <div>
+            <h6>Declaration:</h6><pre>@version: <MadCap:variable name="Version.configversion"></MadCap:variable>
+@include "scl.conf"
+log {
+    source { system(); };
+    parser { mariadb-audit-parser(); };
+    destination { ... };
+};</pre>
+        </div>
+        <p>The <span class="Code" oldrole="parameter">mariadb-audit</span> is actually a reusable configuration snippet configured to parse MariaDB Audit Plugin messages. For details on using or writing such configuration snippets, see <MadCap:xref href="config-blocks.htm"><span style="color: #04aada;" class="mcFormatColor">Reusing configuration blocks</span></MadCap:xref>. You can find the source of this configuration snippet on <a href="https://github.com/syslog-ng/syslog-ng/blob/master/scl/mariadb/audit.conf">GitHub</a>.</p>
```
Comment (tamaskiss00) [inline Content/Guides/syslog-ng-guide-admin/parser-mariadb-audit.htm:26]: oldrole="parameter" not needed

File: Content/Guides/syslog-ng-guide-admin/parser-mariadb-audit.htm [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        <div>
+            <MadCap:snippetBlock src="../shared/chunk/option-parser-prefix.htm">
+            </MadCap:snippetBlock>
+            <MadCap:snippetBlock src="../shared/chunk/para-macro-prefix-parser.htm">
+            </MadCap:snippetBlock>
+            <p>By default, <span class="Code" oldrole="parameter">mariadb-audit</span> uses the <span class="Code" oldrole="userinput">.mariadb.</span> prefix. To modify it, use the following format:</p><pre>parser {
```
Comment (tamaskiss00) [inline Content/Guides/syslog-ng-guide-admin/parser-mariadb-audit.htm:32]: oldrole="parameter" not needed
oldrole="userinput" not needed

File: Content/Guides/syslog-ng-guide-admin/reference-template-functions.htm [hunk 5]
```diff
@@ -559,6 +559,7 @@ <h6 name="template-function-hash"><a name="template-function-hash"></a>hash</h6>
             <p><i style="font-style: normal;">Options:</i><pre>--length N, -l N</pre>Truncate the hash to the first N characters.</p>
             <p><i style="font-style: normal;">Description:</i> Calculates a hash of the string or macro received as argument using the specified hashing method. If you specify multiple arguments, effectively you receive the hash of the first argument salted with the subsequent arguments.</p>
             <p><span class="Code" oldrole="userinput">&lt;method&gt;</span> can be one of md5, md4, sha1, sha256, sha512 and "hash", which is equivalent to md5. Macros are expected as arguments, and they are concatenated without the use of additional characters.</p>
+            <p>The md4 <span class="Code" oldrole="userinput">&lt;method&gt;</span> is deprecated.</p>
```
Comment (tamaskiss00) [inline Content/Guides/syslog-ng-guide-admin/reference-template-functions.htm:562]: oldrole="userinput" not needed

File: Content/Guides/syslog-ng-guide-admin/rewrite-tags.htm [hunk 6]
```diff
@@ -22,6 +22,6 @@ <h1 name="rewrite-tags">Adding and deleting tags</h1>
         <p>To delete a tag, use the following syntax:</p><pre>rewrite &lt;name_of_the_rule&gt; {
     clear-tag("&lt;tag-to-delete&gt;");
 };</pre>
-        <p>You cannot use macros in the tags.</p>
+        <p>Templates (macros, template functions) can be used when specifying tags, for example, <span class="Code" oldrole="userinput">set-tag("dyn::$HOST");</span>.</p>
```
Comment (tamaskiss00) [inline Content/Guides/syslog-ng-guide-admin/rewrite-tags.htm:25]: <span class="Code" oldrole="userinput">set-tag("dyn::$HOST");</span>
instead use <b class="Code">set-tag("dyn::$HOST");</b>

File: Content/Guides/syslog-ng-guide-admin/tlsoptions.htm [hunk 7]
```diff
@@ -97,6 +97,54 @@ <h6 name="tls-options-ecdh-curve-list"><a name="tls-options-ecdh-curve-list"></a
...
+                    </tr>
+                    <tr class="TableStyle-RuledTableWithHeading_DoNotEdit-Body-Body1">
+                        <td class="TableStyle-RuledTableWithHeading_DoNotEdit-BodyB-Column1-Body1">Default:</td>
+                        <td class="TableStyle-RuledTableWithHeading_DoNotEdit-BodyA-Column1-Body1">none </td>
+                    </tr>
+                </tbody>
+                <col class="TableStyle-RuledTableWithHeading_DoNotEdit-Column-Column1" style="width: 0.3in;">
+                </col>
+            </table>
+            <p><i style="font-style: normal;">Description:</i> A colon-separated list that specifies the supported signature algorithms associated with client authentication for TLSv1.2 and higher, for example, <span class="Code" oldrole="userinput">RSA-PSS+SHA256:ed25519</span>.</p>
```
Comment (tamaskiss00) [inline Content/Guides/syslog-ng-guide-admin/tlsoptions.htm:144]: `<span class="Code" oldrole="userinput">RSA-PSS+SHA256:ed25519</span>`
use `<b class="Code">RSA-PSS+SHA256:ed25519</b>` instead

File: Content/Guides/syslog-ng-guide-admin/tlsoptions.htm [hunk 8]
```diff
@@ -97,6 +97,54 @@ <h6 name="tls-options-ecdh-curve-list"><a name="tls-options-ecdh-curve-list"></a
...
+                    </tr>
+                    <tr class="TableStyle-RuledTableWithHeading_DoNotEdit-Body-Body1">
+                        <td class="TableStyle-RuledTableWithHeading_DoNotEdit-BodyB-Column1-Body1">Default:</td>
+                        <td class="TableStyle-RuledTableWithHeading_DoNotEdit-BodyA-Column1-Body1">none </td>
+                    </tr>
+                </tbody>
+                <col class="TableStyle-RuledTableWithHeading_DoNotEdit-Column-Column1" style="width: 0.3in;">
+                </col>
+            </table>
+            <p><i style="font-style: normal;">Description:</i> A colon-separated list that specifies the supported signature algorithms for TLSv1.2 and higher, for example, <span class="Code" oldrole="userinput">RSA-PSS+SHA256:ed25519</span>.</p>
```
Comment (tamaskiss00) [inline Content/Guides/syslog-ng-guide-admin/tlsoptions.htm:120]: `<span class="Code" oldrole="userinput">RSA-PSS+SHA256:ed25519</span>`
use `<b class="Code">RSA-PSS+SHA256:ed25519</b>` instead

File: Content/Guides/syslog-ng-guide-admin/tlsoptions.htm [hunk 9]
```diff
@@ -97,6 +97,54 @@ <h6 name="tls-options-ecdh-curve-list"><a name="tls-options-ecdh-curve-list"></a
...
+            <p>For servers it is used to determine which signature algorithms to support.</p>
```
Comment (tamaskiss00) [inline Content/Guides/syslog-ng-guide-admin/tlsoptions.htm:121]: comma after "For servers"

File: Content/Guides/syslog-ng-guide-admin/tlsoptions.htm [hunk 10]
```diff
@@ -97,6 +97,54 @@ <h6 name="tls-options-ecdh-curve-list"><a name="tls-options-ecdh-curve-list"></a
...
+                    <tr class="TableStyle-RuledTableWithHeading_DoNotEdit-Body-Body1">
+                        <td class="TableStyle-RuledTableWithHeading_DoNotEdit-BodyB-Column1-Body1">Default:</td>
+                        <td class="TableStyle-RuledTableWithHeading_DoNotEdit-BodyA-Column1-Body1">none </td>
+                    </tr>
+                </tbody>
+                <col class="TableStyle-RuledTableWithHeading_DoNotEdit-Column-Column1" style="width: 0.3in;">
+                </col>
+            </table>
+            <p><i style="font-style: normal;">Description:</i> A colon-separated list that specifies the supported signature algorithms associated with client authentication for TLSv1.2 and higher, for example, <span class="Code" oldrole="userinput">RSA-PSS+SHA256:ed25519</span>.</p>
+            <p>For servers the value is used in the <pre>signature_algorithms</pre> field of a <pre>CertificateRequest</pre> message.</p>
```
Comment (tamaskiss00) [inline Content/Guides/syslog-ng-guide-admin/tlsoptions.htm:145]: comma after "For servers"

File: Content/Guides/syslog-ng-guide-admin/tlsoptions.htm [hunk 11]
```diff
@@ -97,6 +97,54 @@ <h6 name="tls-options-ecdh-curve-list"><a name="tls-options-ecdh-curve-list"></a
...
+            <p>For clients it is used to determine which signature algorithm to use with the client certificate.</p>
```
Comment (tamaskiss00) [inline Content/Guides/syslog-ng-guide-admin/tlsoptions.htm:146]: comma after "For clients"

File: Content/Guides/syslog-ng-guide-admin/parser-mariadb-audit.htm (hunk 3 above)
Comment (tamaskiss00) [reply Content/Guides/syslog-ng-guide-admin/parser-mariadb-audit.htm:26]: remove "actually"

File: Content/Guides/syslog-ng-guide-admin/parser-mariadb-audit.htm (hunk 2 above)
Comment (tamaskiss00) [reply Content/Guides/syslog-ng-guide-admin/parser-mariadb-audit.htm:16]: output types' (no 's' needed after the apostrophe)

**Comment by OverOrion** [comment]
Thank you @tamaskiss00 for your notes, I have addressed them all.

**Comment by rakeshbare22** [comment]
hey

**Comment by rakeshbare22** [comment]
can anyone help me..in my one issue

**Comment by MrAnno** [comment]
Please open an issue or raise a question at github.com/syslog-ng/syslog-ng .

**Pull request merged**